src/main/java/org/iot/dsa/dslink/dynamodb/DynamoDBDSAClient.java

```diff
@@ -0,0 +1,146 @@
+package org.iot.dsa.dslink.dynamodb;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.document.DynamoDB;
+import com.amazonaws.services.dynamodbv2.model.ListTablesResult;
+
+import org.iot.dsa.node.DSMap;
+
+import java.util.List;
+
+public class DynamoDBDSAClient {
+
+    private String accessKey = null;
+    private String secretKey = null;
+    private String region = null;
+    private String endpoint = null;
+    private AmazonDynamoDB client;
+
+    public DynamoDBDSAClient(String accessKey, String secretKey, String region, String endpoint){
+        this.accessKey = accessKey;
+        this.secretKey = secretKey;
+        this.region = region;
+        this.endpoint = endpoint;
+        client = Util.connectDynamoDB(accessKey,secretKey,region,endpoint);
+    }
+
+    public void listtable(){
```

Comment:
What is the purpose of this method? Is it just for debugging?

Reply:
it was for debugging. will be removed.

Reply:
It was used for debugging. But using it for displaying tableNames in DB actions.